backend/tools.go

```diff
@@ -11,6 +11,7 @@ import (
 	_ "github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway"
 	_ "github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2"
 	_ "github.com/shurcooL/vfsgen"
-	_ "google.golang.org/protobuf/cmd/protoc-gen-go"
 	_ "google.golang.org/grpc/cmd/protoc-gen-go-grpc"
+	_ "google.golang.org/protobuf/cmd/protoc-gen-go"
+	_ "k8s.io/api"
```

Comment:
```suggestion
```

Reply:
i think this is leftover from the earlier compiling k8s protos attempt

Reply:
hmmm...weird that make backend-verify doesn't clean it up. I'll remove it

Reply:
Ah we add to this only when we don't want `go mod tidy` to clean it up